bin/behat-prepare.sh

```diff
@@ -67,3 +67,18 @@ sleep 10
 ###
 terminus wp "core install --title=$TERMINUS_ENV-$TERMINUS_SITE --url=$PANTHEON_SITE_URL --admin_user=pantheon --admin_email=wp-redis@getpantheon.com --admin_password=pantheon"
 terminus wp "plugin activate wp-native-php-sessions"
+
+###
+# Download the Pantheon WordPress Upstream tests
+###
+cd $BASH_DIR/..
+rm -rf pantheon-wordpress-upstream-master tests/pantheon-wordpress-upstream
+wget https://github.com/pantheon-systems/pantheon-wordpress-upstream/archive/master.zip
```

Comment:
I'm going to switch this downloading to composer as I did in https://github.com/pantheon-systems/wp-redis/pull/124

I'll commit directly to the branch of this PR rather than making another new one.